setup.py

```diff
@@ -428,7 +428,29 @@ def macos_extensions():
     ]
 
 
+def verify_swig_version():
+    """ Verify we are using swiig version 3.0.x.
+    """
+    msg = ("SWIG is a required build dependency of Enable. Furthermore, there "
+          "is currently a known issue with SWIG 4.0, see enthought/enable#360."
+          " Please install SWIG 3.0.x (see http://www.swig.org/).")
+    try:
+        with subprocess.Popen(["swig", "-version"], \
+                stdout=subprocess.PIPE, encoding='utf-8') as proc:
+            # We expect text along the lines of "SWIG Version 3.X.Y"
+            swig_version_match = re.search(
+                '(SWIG Version 3)\.\d{1,2}\.\d{1,2}',
```

Comment:
Shouldn't this be `r"\(SWIG Version 3\)\.\d{1,2}\.\d{1,2}"`? Did you test this?

Reply:
I tested and everything seemed to work as I had expected (I got an error with swig 4, rolled back to swig 3 and install succeeded)
why the `\(`, and `\)` as we don't want literal "(" or ")"?
The leading r is probably safer, I think I may have just lucked out here that it works without it 
I will update